modules/aws-outposts-ingress-controller-service.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-service_{context}"]
+= Configuring the default Ingress Controller in an AWS VPC cluster extended into an Outpost
+
+Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost if edge and cloud-based subnets are in the same availability zone.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
```

Comment:
I'm struggling to understand these phrases. The Classic Load Balancer can target (send traffic to) cloud-based and outpost instances (nodes). This way if the workload (the final destination of the traffic) is scheduled on a cloud based or outpost node by OpenShift the CLB can still reach it. This introduction is supposed to tell the user that CLB can load balance between the mixed locales (cloud, outpost) and the follow up phrases are supposed to tell the user that the ingresscontroller can be scheduled on mixed instances too. So they can work as is, without any additional configurations. But to avoid the latency (which is likely to be present) between the cloud based locale and outpost locale a procedure can be followed.